src/battle/area/kmr_part_1/stage/kmr_bt02.cpp

```diff
@@ -0,0 +1,28 @@
+#include "../area.hpp"
+
+namespace battle::area::kmr_part_1 {
+
+namespace kmr_02 {
+
+EvtScript EVS_PreBattle = {
+    Call(SetSpriteShading, SHADING_NONE)
+    Return
+    End
+};
+
+EvtScript EVS_PostBattle = {
+    Return
+    End
+};
+
+} // namespace kmr_02
```

Comment:
Since these symbols aren't needed outside of the file, put everything but the Stage in an [anonymous namespace](https://www.learncpp.com/cpp-tutorial/unnamed-and-inline-namespaces/), and just leave the stage accessible under the `battle::area::kmr_part_1` namespace.